addon/-private/system/relationships/state/has-many.js

```diff
@@ -12,6 +12,25 @@ export default class ManyRelationship extends Relationship {
     this.belongsToType = relationshipMeta.type;
     this.canonicalState = [];
     this.isPolymorphic = relationshipMeta.options.polymorphic;
+    this._manyArray = null;
+    this.__loadingPromise = null;
+  }
+
+  get _loadingPromise() { return this.__loadingPromise; }
```

Comment:
Wanted to make sure it was readOnly without going through the update method. (So people don't accidentally bypass)